src/store/journal/journal.actions.ts

```diff
@@ -0,0 +1,25 @@
+import { Action } from '@ngrx/store';
+
+export const LOAD_JOURNAL = '[Main] Load Journal';
```

Comment:
`JournalPage` is the initiator of this action. Let's use that name in the square brackets.